create_openapi_definition.py

```diff
@@ -0,0 +1,216 @@
+"""
+create_openapi_definition.py.
+
+This script is used to create a full OpenAPI v3 definition file in yaml, which
+is then uploaded to Aportio's Swaggerhub profile and can be found at
+<insert-aportio-swaggerhub-link-here>.
```

Comment:
Will this link be added as part of this pr??

Reply:
Yes, just added it now since Juergen created our Aportio account